backend/integrations/plain.py

```diff
@@ -155,19 +137,27 @@ def _send_chat(customer_id: str, title: str, message: str):
         document,
         variables={
             "input": {
-                "customerId": customer_id,
                 "title": title,
+                "customerIdentifier": {"customerId": customer_id},
                 "components": [{"componentText": {"text": message}}],
             }
         },
     )
 
-    _raise_mutation_error(response, "upsertCustomTimelineEntry")
+    _raise_mutation_error(response, "createThread")
+    thread_id = response["createThread"]["thread"]["id"]
 
-    logger.info("Custom timeline entry added on Plain with title '%s'", title)
+    logger.info("Thread created with id: %s", thread_id)
+    return thread_id
 
 
 def send_message(user: User, title: str, message: str):
     customer_id = create_customer(user)
-    change_customer_status(customer_id)
-    _send_chat(customer_id, title, message)
+    thread_id = _create_thread(customer_id, title, message)
+
+    try:
```

Comment:
This is a bit of mix of concerns, the plain integration doesn't/shouldnt know anything about the grant

Say we want to re-use this function to send some other messages to Plain

Reply:
yeah I see